src/region.js

```diff
@@ -86,7 +86,7 @@ const Region = MarionetteObject.extend({
 
   _attachView(view, {replaceElement} = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!replaceElement || !!this.getOption('replaceElement');
+    const shouldReplaceEl = replaceElement === true || !!this.getOption('replaceElement');
```

Comment:
:-) my bad.. this doesn't work either..  really we probably want `options` and `_.default`.  if you've explicitly set `replaceElement` to false but it is `true` on the obj, it would still end up true.  Can't do `||`